src/components/Person/Person.jsx

```diff
@@ -1 +1,35 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const sexPartner = (person.sex === 'm') ? 'wife' : 'husband';
```

Comment:
Add clarifying keyword to your var name
```suggestion
  const sexPartnerStatus = (person.sex === 'm') ? 'wife' : 'husband';
```